examples/annotation_import/video.ipynb

```diff
@@ -1,346 +1,1016 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    frame_bbox_with_checklist_subclass_ndjson = {
</pre><p>Can you just make a note here as to what we are doing. Just mention that we do not support classifications by name and that we will populate these later in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done !</p>